capreolus/searcher/__init__.py

```diff
@@ -50,7 +51,8 @@ def _anserini_query_from_file(self, topicsfn, anserini_param_str, output_base_pa
             logger.debug(f"skipping Anserini SearchCollection call because path already exists: {donefn}")
             return
 
-        self["index"].create_index()
+        if not self["index"].exists():
+            raise Exception("Index not build")
```

Comment:
TODO: Replace this generic exception with a `IndexNotBuild` exception once I pull from @Crysitna 's changes